textacy/datasets/wikipedia.py

```diff
@@ -420,12 +422,13 @@ def records(self, min_len=100, limit=-1, fast=False):
 
         n_pages = 0
         for page_id, title, content in self:
-            page = self._parse_content(content, parser, fast)
+            page = self._parse_content(content, parser, fast, keep_section_headings)
             if len(page['text']) < min_len:
                 continue
             page['title'] = title
             page['page_id'] = page_id
-            page['text'] = title + '\n\n' + page['text']
+            if keep_section_headings:
```

Comment:
Looks like this only handles the case when `keep_section_headings` is True. You'll need an `else` statement for the other case.

Reply:
will do

Reply:
after looking at this, page['text'] already has it's value, we only want to modify it's value if we want to prepend the title, so unless I'm missing something we don't need an else